diff --git a/node_modules/release/bin/release.js b/node_modules/release/bin/release.js
index b005c99..c67a1e7 100755
--- a/node_modules/release/bin/release.js
+++ b/node_modules/release/bin/release.js
@@ -39,6 +39,7 @@ args.option('pre', 'Mark the release as prerelease')
 	.option(['H', 'hook'], 'Specify a custom file to pipe releases through')
 	.option(['t', 'previous-tag'], 'Specify previous release', '')
 	.option(['u', 'show-url'], 'Show the release URL instead of opening it in the browser')
+  .option(['x', 'prefix-tag'], 'Prefix tag name with "v"', false)
 	.option(['s', 'skip-questions'], 'Skip the questions and create a simple list without the headings');
 
 const flags = args.parse(process.argv);
@@ -95,7 +96,7 @@ const createRelease = async (tag, changelog, exists) => {
 		owner: repoDetails.user,
 		repo: repoDetails.repo,
 		/* eslint-disable camelcase */
-		tag_name: tag.tag,
+    tag_name: flags.prefixTag ? `v${tag.tag}` : tag.tag,
 		target_commitish: tag.hash,
 		/* eslint-enable camelcase */
 		body: changelog,
